### PR TITLE
mc_pos_control: replace the dt by  global variable _dt

### PR DESCRIPTION
**Pull request by huangwen0907**
@MaEtUgR 
I am more prefer the global `_dt`, please have a review

### CODE REVIEW

**Comment by dagar** [comment]
It's not very obvious, but because MulticopterPositionControl is a control:SuperBlock (https://github.com/PX4/Firmware/blob/master/src/lib/controllib/block/Block.hpp#L101) it already has a _dt (https://github.com/PX4/Firmware/blob/master/src/lib/controllib/block/Block.hpp#L94).

It's already set here. https://github.com/PX4/Firmware/blob/master/src/modules/mc_pos_control/mc_pos_control_main.cpp#L3049

File: src/modules/mc_pos_control/mc_pos_control_main.cpp [hunk 1]
```diff
@@ -295,6 +295,7 @@ class MulticopterPositionControl : public control::SuperBlock
 	float _manual_jerk_limit_xy; /**< jerk limit in manual mode dependent on stick input */
 	float _manual_jerk_limit_z; /**< jerk limit in manual mode in z */
 	float _z_derivative; /**< velocity in z that agrees with position rate */
+	float _dt;			/* record the controller update rate*/
```
Comment (dagar) [inline src/modules/mc_pos_control/mc_pos_control_main.cpp]: Use _dt from Block base class (https://github.com/PX4/Firmware/blob/master/src/lib/controllib/block/Block.hpp#L94).

Comment (huangwen0907) [reply src/modules/mc_pos_control/mc_pos_control_main.cpp]: @dagar  Thanks for your comments. Add a new https://github.com/PX4/Firmware/pull/8651/commits/5927468510b3eab15e77e29d55f778f74f6811d5

Comment (dagar) [reply src/modules/mc_pos_control/mc_pos_control_main.cpp]: @huangwen0907 what I meant was you can delete this line entirely and still use `_dt` in mc_pos_control.

Comment (huangwen0907) [reply src/modules/mc_pos_control/mc_pos_control_main.cpp]: @dagar Ah~ Just now I understand what you mean. yes, what you said is much better.
Do you think this pr is necessary? if need, I can reopen, if not, just forget it.

Comment (dagar) [reply src/modules/mc_pos_control/mc_pos_control_main.cpp]: Up to you. If you delete that _dt the rest should be ready to go.

**Pull request closed without merging**